Tests/regressionTests.js

```diff
@@ -0,0 +1,301 @@
+        .then(() => registerPage.enterEmail(email))
+        .then(() => registerPage.enterPassword(password))
+        .then(() => registerPage.enterConfirmPassword(password))
+        .then(() => registerPage.clickOnRegisterButton())
+        .then(() => browser.wait(EC.presenceOf($(".status-success")), 5000))
+        .then(() => registerPage.clickOnHereLink())
+        .then(() => browser.wait(EC.presenceOf($(".form-title")), 5000))
+        .then(() => loginpage.enterEmail(email))
+        .then(() => loginpage.enterPassword(password))
+        .then(() => loginpage.clickOnLoginButton())
+        .then(() => browser.wait(EC.presenceOf($(".have-account")), 5000))
+        .then(() => registerPage.getRegisterURL())
+        .then((data) => expect(data).toBe("https://auct-app2-frontend.herokuapp.com/register"))
+        }); 
+
+    it("002: User is able to log in from 'Register' page", function(){
+        var firstName="$";
+        var lastName="$";
+        var email = `wok146${random1}${random2}@mail.com`;
+        var password ="%%%%%";
+        homepage.clickOnCreateAnAccont()
+        .then(() =>  registerPage.enterFirstName(firstName))
+        .then(() => registerPage.enterLastName(lastName))
+        .then(() => registerPage.enterEmail(email))
+        .then(() => registerPage.enterPassword(password))
+        .then(() => registerPage.enterConfirmPassword(password))
+        .then(() => registerPage.clickOnRegisterButton())
+        .then(() => browser.wait(EC.presenceOf($(".status-success")), 5000))
+        .then(() => registerPage.clickOnWordLogin())
+        .then(() => browser.wait(EC.presenceOf($(".form-title")), 5000))
+        .then(() => loginpage.enterEmail(email))
+        .then(() => loginpage.enterPassword(password))
+        .then(() => loginpage.clickOnLoginButton())
+        .then(() => browser.wait(EC.presenceOf($(".have-account")), 5000))
+        .then(() => registerPage.getRegisterURL())
+        .then((data) => expect(data).toBe("https://auct-app2-frontend.herokuapp.com/register"))
+    })
+    it ("003: User is able to log in from 'Create an account' form", function(){
+        var userName = "rafa@mail.com";
+        var password = "password";
+        homepage.clickOnCreateAnAccont()
+        .then (() =>homepage.clickOnLogin())
+        .then(() => loginpage.enterEmail(userName))
+        .then(() => loginpage.enterPassword(password))
+        .then(() => loginpage.clickOnLoginButton())
+        .then(() => browser.wait(EC.presenceOf($(".have-account")), 5000))
+        .then(() => loginpage.getLoginURL())
+        .then((data) => expect(data).toBe("https://auct-app2-frontend.herokuapp.com/register"))
+    }) 
+    it ("004: User isn't able to log in without password", function(){
+        var userName = "rafa@mail.com"
+        homepage.clickOnLogin()
+        .then(() => loginpage.enterEmail(userName))
+        .then(() => loginpage.clickOnLoginButton())
+        .then(() => browser.sleep(2000))
```

Comment:
try not to use browser.sleep in code

Reply:
I added this, because there is no dynamic loading in HTML, so I don't have what to wait for, I'll try to communicate this with Developer